frontend/src/premium/user/permission-checker.js

```diff
@@ -105,11 +105,11 @@ export class PermissionChecker {
 
   get currentTenantPlan() {
     if (!this.currentTenant) {
-      return plans.free
+      return plans.essential
```

Comment:
Are these the only changes we need to include related to plans?
We also have the `plans.js` file, which we should also update I think

Reply:
Sorry I forgot to mention but both permissions and plans file were updated in Anil's PR https://github.com/CrowdDotDev/crowd.dev/pull/318/files#diff-0e09062fc2c86ce9490bfbd2117e20613eb73e9cb7e7c7232a879213e76696d6

This one cannot be deployed without that PR. Given that this branch does not include backend changes, to test it locally you will either need to test it in staging branch or run with the backend from the `feature/tenant-plans-and-feature-flagging` branch